setup/engine-templates/base-suricata.template

```diff
@@ -2,7 +2,7 @@
 ---
 host-mode: auto
 unix-command:
-  enabled: yes
+  enabled: no
```

Comment:
Wondering if we need to also add `rule-files:` to the bottom of template as well?

Reply:
Engine specific settings get appended to template, ya? if so, this will cause test-test ruleset to fail because it appears the rule file name is appended to config

Reply:
What I meant was that - each engine specific template is "derived" from a [base template](https://github.com/EmergingThreats/IDSDeathBlossom/blob/master/setup/engine-templates/base-suricata.template) (example) upon the engines installation/addition. So from that perspective it may make sense to do it ?